### PR TITLE
Added one color theme

### PR DESCRIPTION
**Pull request by ZoltanDalmadi**
Added matching theme for [one](https://github.com/rakr/vim-one), which gained popularity as the default color theme of the [Atom](https://atom.io) text editor.

Dark version:
lightline_one_dark

Light version:
lightline_one_light

### CODE REVIEW

**Comment by itchyny** [comment]
I prefer the both files merged into one file like #175.

**Comment by ZoltanDalmadi** [comment]
Merged them into one file.

File: autoload/lightline/colorscheme/one.vim [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+	let s:gray3  = '#3e4452'
+
+	let s:p.normal.left     = [ [ s:bg, s:green, 'bold' ], [ s:fg, s:gray3 ] ]
+	let s:p.normal.middle   = [ [ s:fg, s:gray2 ] ]
+	let s:p.inactive.left   = [ [ s:gray1,  s:bg ], [ s:gray1, s:bg ] ]
+	let s:p.inactive.middle = [ [ s:gray1, s:gray2 ] ]
+	let s:p.inactive.right  = [ [ s:gray1, s:bg ], [ s:gray1, s:bg ] ]
+	let s:p.insert.left     = [ [ s:bg, s:blue, 'bold' ], [ s:fg, s:gray3 ] ]
+	let s:p.replace.left    = [ [ s:bg, s:red1, 'bold' ], [ s:fg, s:gray3 ] ]
+	let s:p.visual.left     = [ [ s:bg, s:purple, 'bold' ], [ s:fg, s:gray3 ] ]
```
Comment (itchyny) [inline autoload/lightline/colorscheme/one.vim]: prefer soft tabs (2 spaces)

**Comment by ZoltanDalmadi** [comment]
Changed indentation to soft tabs (2 spaces).

**Comment by itchyny** [comment]
Thank you for your contribution.

**Pull request merged**